script/cache-hash.sh

```diff
@@ -1,10 +1,21 @@
 #!/usr/bin/env bash
 
-JNI_FILES="magic.txt app/build.gradle app/src/main/jni/cmake/* app/src/main/jni/librime_jni/* app/src/main/jni/CMakeLists.txt"
+DIGEST_ALGORITHM=sha1sum
```

Comment:
https://en.wikipedia.org/wiki/SHA-1 SHA-1 is insecure, SHA-256 is the [default algorithm of Github Action](https://docs.github.com/en/actions/learn-github-actions/expressions#hashfiles)